### ISSUE TITLE
"Content is not within a landmark element" for overflow menu items

### ISSUE THREAD

**Issue by sjhowell** [carbon-design-system/carbon#8078]
## Brief description

"Content is not within a landmark element" is reported by the "IBM Equal Access Accessibility Checker" plugin for the buttons of  `<OverflowMenuItem>` components within an `<OverflowMenu>`.

## Environment

- Operating system: Windows and Mac
- Browser: Firefox and Chrome
- Automated testing tool and ruleset: IBM Accessibility Equal Access Toolkit
- Assistive technology used to verify: IBM Accessibility Equal Access Toolkit

## Detailed description

We have an `<OverflowMenu>` on our page which contains `<OverflowMenuItem>` components. With the overflow menu open and the items visible, the IBM Accessibility Equal Access Toolkit raises the following violation for this:

![image](https://user-images.githubusercontent.com/40432849/110946716-23ae5680-8337-11eb-8cc7-4b48be32ba69.png)

- What version of the Carbon Design System are you using? 7.25.0
- What did you expect to happen? No accessibilty violation to occur
- What happened instead? Accessibility violation occurred
- What WCAG 2.1 checkpoint does the issue violate? "2.4.1 Bypass blocks"

**Comment by emyarod** [comment]
just as the violation details state, the solution for this is to have a `<main>` element enclosing the content of your page or placing `role="main"` on the container element. for example in the Carbon storybook, the component is contained in a `<div role="main">` element to avoid this violation
